src/ReactPlayer.js

```diff
@@ -42,7 +42,7 @@ export default class ReactPlayer extends Component {
     showPreview: !!this.props.light
   }
 
-  refs = {
+  elementRefs = {
```

Comment:
`refs` overshadows legacy [`React.Component`](https://reactjs.org/docs/refs-and-the-dom.html#legacy-api-string-refs) prop and is overwritten.